### PR TITLE
feat: add rewards-v2 support to staker-operator table generation

### PR DESCRIPTION
**Pull request by seanmcgary**

### CODE REVIEW

**Review by 0xrajath — DISMISSED** [review]
LGTM

File: pkg/rewards/stakerOperators/8_avsODStrategyPayouts.go
Comment (0xrajath) [inline pkg/rewards/stakerOperators/8_avsODStrategyPayouts.go:1]: Just realised something. Should we dedupe this across strategies? I think in rewards calculation the staging step does a distinct that takes care of it.

Comment (0xrajath) [reply pkg/rewards/stakerOperators/8_avsODStrategyPayouts.go:1]: Or do the same distinct filtering step in the stakerOperatorStaging step.

Comment (seanmcgary) [reply pkg/rewards/stakerOperators/8_avsODStrategyPayouts.go:1]: We could make this a `sum(amount)` where its grouped by `avs, operator, strategy, reward_hash, snapshot`, but I think that should already be the case....

**Pull request merged**